content/docs/05.developer-guide/01.flow.md

```diff
@@ -77,6 +77,45 @@ The following flow properties can be set.
 |`taskDefaults.[].values.xxx`| The task property that you want to be set as default.                                                                                                                                        |
 |`disabled`| Set it to `true` to disable execution of the flow.                                                                                                                                           |
 
+
+### `taskDefaults`
+
+`taskDefaults` avoid to repeat task properties. For example:
```

Comment:
```suggestion
>You can add task defaults to avoid repeating task properties on multiple occurrences of the same task in a `taskDefaults` properties. For example:
```

I also prefer valid sentences in documentation, it's easier to read.